utilities/src/main/kotlin/org/jetbrains/kotlin/cli/utilities/GeneratePlatformLibraries.kt

```diff
@@ -10,6 +10,8 @@ import org.jetbrains.kotlin.konan.file.File
 import java.util.concurrent.*
 import kotlinx.cli.*
 
+const val platformPrefix = "org.jetbrains.konan.platform."
```

Comment:
(minor) May be move this constant into the `shared` project? In this case it will be available in both `platformLibs/build.gradle` and `GeneratePlatformLibraries.kt`.

Reply:
`konan` -> `kotlin.native`?

Reply:
Not sure if using java keyword is a good idea here.